src/storage/errors.rs

```diff
@@ -318,16 +318,21 @@ pub fn extract_region_error_from_error(e: &Error) -> Option<errorpb::Error> {
             Some(err)
         }
         Error(box ErrorInner::DeadlineExceeded) => {
-            let mut err = errorpb::Error::default();
-            let mut server_is_busy_err = errorpb::ServerIsBusy::default();
-            server_is_busy_err.set_reason(DEADLINE_EXCEEDED.to_owned());
-            err.set_server_is_busy(server_is_busy_err);
+            let err = make_deadline_exceeded_busy_error();
             Some(err)
         }
         _ => None,
     }
 }
 
+pub fn make_deadline_exceeded_busy_error() -> errorpb::Error {
```

Comment:
This could also be moved to the `deadline.rs` in the `tikv_util` component so there is no duplication.